climakitae/threshold_tools.py

```diff
@@ -49,6 +61,9 @@ def get_ams(da, extremes_type="max"):
 
 
 #####################################################################
+# GET L-MOMENTS DISTR
+# input: distribution name
+# export: corresponding l-moments distribution function
```

Comment:
While this formatting looks nice, I think it's better to put blocks of 
'''
This kind of comment
'''
just after function def lines, because that standard format will get grabbed to create automatic documentation, which #comments will not.

Reply:
updated function comments to this formatting. thanks for catching this and explaining!